dashboard/test/scripts/update_tts_i18n_test.rb

```diff
@@ -32,7 +32,7 @@ class UpdateTtsI18nTest < ActiveSupport::TestCase
     create :multi, name: 'contained level', long_instructions: "contained level long instructions"
     level = create :maze, name: 'containing level', contained_level_names: ['contained level']
 
-    level.expects(:tts_upload_to_s3).with("contained level long instructions\n")
+    level.expects(:tts_upload_to_s3).with {|text, _| text == "contained level long instructions\n"}
```

Comment:
any reason not to just do
```suggestion
    level.expects(:tts_upload_to_s3).with("contained level long instructions\n", "update_level_i18n")
```

Reply:
that's a great point; I think the tradeoff here is in code complexity and readability. I'd suggest adding a comment explaining basically what you just explained to me :)